firmware/doc.txt

```diff
@@ -4,5 +4,5 @@
 # What goes here?
 Specific documentation for our reference firmware ; m4a-firmware
 
-Add more: specific documentation for the m4a-firmware
+Add more: specific documentation for the m4a-firmware scren
 */
```

Comment:
I know that is an error. Take a look to the errors on static-test and you will see that I reffer